### DISCUSSION TITLE
Headers in content tabs

### DISCUSSION THREAD

**Discussion by izzybroderick** [squidfunk/mkdocs-material#3981] [Q&A] [answered]
Is it possible to navigate to a specific header using the right-hand TOC when it's nested in a content tab?

In the below example, all four headers appear in the right-hand table of contents. In the TOC, if I click "Deploying models to production," is there a way to have this open `tab2` and bring the reader to that specific header?

```
=== "tab1"

    ## Importing data
    ## Transforming data

=== "tab2"

    ## Analyzing models
    ## Deploying models to production

```

**Comment by squidfunk** [comment, marked as answer]
No, currently not possible. We're working on bringing the reader to the active tab, but you shouldn't use headers in content tabs, as they're not visible when the content tab is not selected which makes them not targettable through the table of contents.

- #3527
- #3879
- #3976
